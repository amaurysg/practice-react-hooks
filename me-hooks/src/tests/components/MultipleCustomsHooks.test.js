import React from "react";
import { shallow } from "enzyme";
import MultipleCustomsHooks from "../../components/examples/MultipleCustomsHooks";

describe("Test in component MultipleCustomsHooks", () => {
  test(" Debe de mostrarse correctamente ", () => {
    const wrapper = shallow(<MultipleCustomsHooks />);
    expect(wrapper).toMatchSnapshot();
  });
});
