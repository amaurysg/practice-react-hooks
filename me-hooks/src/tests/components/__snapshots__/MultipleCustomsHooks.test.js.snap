// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in component MultipleCustomsHooks  Debe de mostrarse correctamente  1`] = `
<div>
  <h1>
    Breaking Bad Quotes
  </h1>
  <hr />
  <div
    className="alert alert-info text-center"
  >
    Loading...
  </div>
  <button
    className="btn btn-primary"
    onClick={[Function]}
  >
    Next quote
  </button>
</div>
`;
